src/kili/services/export/format/kili/__init__.py

```diff
@@ -50,9 +50,8 @@ def _save_assets_export(self, assets: List[Asset], output_filename: Path):
                 images_folder = base_folder / "images"
                 images_folder.mkdir(parents=True)
                 for asset in assets:
-                    img_path = images_folder / f'{asset["externalId"]}.jpg'
                     if os.path.isfile(asset["content"]):
-                        shutil.move(asset["content"], img_path)
+                        shutil.move(asset["content"], images_folder)
```

Comment:
I did not see it the first time, but we should do a copy since we do not want to remove the files if they have been downloaded through a user call to `.assets`.
The nicest way to do that would be specify the download folder directly in the `.assets` call in the export implementation.

Reply:
👍